index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./CSS/style.css">
+    <title>Wara Conference</title>
+</head>
+<body>
+    <header>
+        <div class="only-desktop">
+            <a href="#">
+                <img src="./Images/awcflogo.png" alt="Logo of AWCF">
+            </a>
+            <nav></nav>
+        </div>
+        <div class="only-mobile hamburger-icon">
+            <a href="html/about.html">
+                <img src="./Images/hamburger-icon.svg" alt="">
+            </a>
+        </div>
+    </header>
```

Comment:
According to the project requirement, Kindly add this interaction for the mobile menu 👍 
 
  - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
  - There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
The mobile menu should have a close (X) button that closes the menu.

Reply:
Resolved